csrc/host_ir/lower.cpp

```diff
@@ -591,6 +591,23 @@ std::vector<Expr*> HostIrLower::lowerToCollectiveBasedPipelinedGemmComm(
       get_current_stream, allocate_tva_allgathered, allocate_tv_out, for_loop};
 }
 
+bool HostIrLower::isLoweredAsStandaloneHostOp(Expr* expr) {
```

Comment:
```suggestion
bool HostIrLower::isLowerableAsStandaloneHostOp(Expr* expr) {
```
? I assume `expr` is still in fusion IR that hasn't been lowered yet.